docs/basics/sso/setting-up-sso.md

```diff
@@ -20,14 +20,22 @@ Sauce Labs supports Identity Provider (IdP)-initiated and Service Provider (SP)-
 First, set up a new SAML application in your [Identity Provider (IdP)](#setting-up-identity-provider). Next, enable and configure SSO in [Sauce Labs Team Management Panel](#integrating-with-sauce-labs-service-provider).
 
 :::note
-SSO at Sauce Labs is global. In other words, it does not matter in which region 
+SSO at Sauce Labs is global. In other words, it does not matter in which region
 you set it up. Unlike to the [old SSO implementation](/basics/sso-deprecated/setting-up-single-sign-on), which has been deprecated.
 :::
 
 ## Setting Up Identity Provider
 
+:::tip
+We provide ready-made SAML applications for some identity providers. They allow you to set up SSO integration with just a few clicks. Choose your provider and follow the configuration tutorial:
+
+- [OneLogin](/basics/sso/configuring-sso-in-onelogin)
+  :::
+
+If you use a custom identity provider or your provider does not appear in the list above, please follow the tutorial for manual setup:
+
 1. Obtain SAML metadata from Sauce Labs Service Provider, which is served under [this link](https://accounts.saucelabs.com/am/sso/metadata/https%3A%2F%2Faccounts.saucelabs.com%2Fsp).
-2. Log in to your IdP admin account, such as Okta, OneLogin, or Microsoft Azure Active Directory, etc.
+2. Log in to your identity provider administrator panel.
```

Comment:
IMO it would increase clarity if we provide example, even one will suffice 
`Okta, OneLogin etc.`

Reply:
fixed